Base/usr/share/man/man1/su.md

```diff
@@ -0,0 +1,41 @@
+## Name
+
+su - run a command with substitute user
+
+## Synopsis
+
+```
+$ su [user]
+```
+
+## Description
+
+**su** allows commands to be run with a substitute user.
+
+When called with no user-specified, **su** defaults to running an interactive shell as *root*. 
```

Comment:
"user specified"

Also, as I said above, su switches the user, so it always runs the shell for some user. Please change the second sentence to specify that the default user for su is root.